src/ses.js

```diff
@@ -134,18 +134,18 @@ export class SesFormation {
     };
   }
 
-  async resourcesFormation() {
+  async resourcesFormation(stage) {
     return {
       VisitorConfirmationTemplate: await this.templateFormation({
-        name: "visitorConfirmationTemplate",
+        name: `visitorConfirmationTemplate-${stage}`,
```

Comment:
In other resources `stage` is prefix so I see some inconsistency here.

Reply:
Done.